src/underactuated/pyplot_visualizer.py

```diff
@@ -51,6 +47,42 @@ def _DoPublish(self, context, event):
     def draw(self, context):
         print "SUBCLASSES MUST IMPLEMENT."
 
+    def animate(self, log, resample=True, repeat=False):
+        # log - a reference to a pydrake.systems.primitives.SignalLogger that
+        # contains the plant state after running a simulation.
+        # rate - the frequency of frames in the resulting animation
```

Comment:
zap this? (or move to a function comment for the constructor?)

Reply:
done